fetch-server/config.py

```diff
@@ -11,3 +13,8 @@
 
 MAPBOX_ACCESS_TOKEN = 'pk.eyJ1IjoiZmV0Y2gwMDEiLCJhIjoiY2o5Z2VwYTVxMnY0dTMzcndibGlrY242dyJ9.0it-uEXRGgddk0XGHVbcgg'
 
+
+# FS_IMAGES_ROOT = '~/fetch_fs/images'
+FS_IMAGES_ROOT = os.path.join(os.getenv('HOME'), 'fetch_fs', 'images')
```

Comment:
Lets make this a relative path to the repo with `os.getcwd`, i believe. That way images have their own folder inside the repo and not in everyones home folder. Then lets gitignore the folder.